src/commands/install-gem-dependencies.yml

```diff
@@ -0,0 +1,20 @@
+description: >
+  Restores, installs and caches Gems
+parameters:
+  cache-prefix:
+    type: string
+    default: ""
+  cache-version:
+    type: string
+    default: v1
```

Comment:
Should we remove the defaults from here and make it a mandatory parameter? (Honestly not sure if that's possible)

Reply:
Let me check, I like it